packages/runner/test/cell.test.ts

```diff
@@ -22,63 +22,45 @@ describe("Cell", () => {
   });
 
   it("should create a cell with initial value", () => {
-    const c = runtime.documentMap.getDoc(
-      10,
-      "should create a cell with initial value",
-      "test",
-    );
+    const c = runtime.getCell<number>("test", "should create a cell with initial value");
+    c.setRaw(10);
```

Comment:
in this case `c.set` would work as well, let's prefer `setRaw` when we intent to do something that requires skipping that layer.